city_scrapers/spiders/chi_northwest_home_equity.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from city_scrapers_core.constants import BOARD, COMMISSION, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthwestHomeEquitySpider(CityScrapersSpider):
+    name = "chi_northwest_home_equity"
+    agency = "Chicago Northwest Home Equity Assurance Program"
+    timezone = "America/Chicago"
+    start_urls = ["https://nwheap.com/category/meet-minutes-and-agendas/"]
+
+    def parse(self, response):
```

Comment:
Looking at this overall, I think it would be best if we could scrape each meeting URL separately. It seems like they don't generally have much information on the detail page, but at the very least we'll be able to get a unique URL for each meeting's `source`, and in some cases they've included dial-in information and links like this page https://nwheap.com/2020/11/30/december-10-2020-meeting-minutes-and-agenda-annual-budget-meeting/

[`chi_ssa_25`](https://github.com/City-Bureau/city-scrapers/blob/c42620480f9184ca74885e2a3c6ef576faba3eb7/city_scrapers/spiders/chi_ssa_25.py) might be a good example to go off of for this. `parse` should yield requests to each visible detail page, and a separate method like `_parse_detail` can yield a single `Meeting` for each detail page